src/reducers/filterMarkers.js

```diff
@@ -27,6 +27,10 @@ const initialState = {
     openNow: false,
     accessTypesHidden: []
   },
+  bathroomFilters: {
+    // TODO
+  },
+  foragingFilters: {},
```

Comment:
If you want, we really don't actually need these with the new code I'll be adding, but feel free to keep them and I can always remove it as I do my migration.